api/tests/integration/domain/services/pick-certification-challenge_test.js

```diff
@@ -0,0 +1,603 @@
+                        statut: '', // unoperative
+                        langues: ['Franco Français']
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ];
+
+    const airtableObjects = airtableBuilder.factory.buildLearningContent(learningContent);
+    airtableBuilder.mockLists(airtableObjects);
+
+    await _buildCorrectAnswerAndKnowledgeElement({
+      userId: certifiableUserId,
+      competenceId: 'recArea1_Competence1',
+      skillId: 'recArea1_Competence1_Tube1_Skill1',
+      challengeId: 'recArea1_Competence1_Tube1_Skill1_Challenge1',
+      pixValue: sufficientPixValueToBeCertifiableOnCompetence,
+      acquisitionDate: placementDate,
+    });
+    const placementProfile = await placementProfileService.getPlacementProfile({
+      userId: certifiableUserId,
+      limitDate: certificationDate
+    });
+
+    // when
+    const challenges = await certificationChallengesService.pickCertificationChallenges(placementProfile);
+
+    //then
+    expect(challenges.map((challenge) => challenge.challengeId)).to.be.empty;
+  });
+
+  it('picks no challenge other than fr-fr', async () => {
+    // given
+    const learningContent = [
+      {
+        id: 'recArea1',
+        competences: [
+          {
+            id: 'recArea1_Competence1',
+            tubes: [
+              {
+                id: 'recArea1_Competence1_Tube1',
+                skills: [
+                  {
+                    id: 'recArea1_Competence1_Tube1_Skill1',
+                    status: 'actif',
+                    challenges: [
+                      {
+                        id: 'recArea1_Competence1_Tube1_Skill1_Challenge1',
+                        statut: 'validé',
+                        langues: ['Anglais', 'Français']
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ];
+
+    const airtableObjects = airtableBuilder.factory.buildLearningContent(learningContent);
+    airtableBuilder.mockLists(airtableObjects);
+
+    await _buildCorrectAnswerAndKnowledgeElement({
+      userId: certifiableUserId,
+      competenceId: 'recArea1_Competence1',
+      challengeId: 'recArea1_Competence1_Tube1_Skill1_Challenge1',
+      pixValue: sufficientPixValueToBeCertifiableOnCompetence,
+      acquisitionDate: placementDate
+    });
+    await databaseBuilder.commit();
+    const placementProfile = await placementProfileService.getPlacementProfile({
+      userId: certifiableUserId,
+      limitDate: certificationDate
+    });
+
+    // when
+    const challenges = await certificationChallengesService.pickCertificationChallenges(placementProfile);
+
+    //then
+    expect(challenges.map((challenge) => challenge.challengeId)).to.be.empty;
+  });
+
+  it('picks no challenge on uncertifiable competences', async () => {
```

Comment:
Surtout dans ce "it" ci les deux négations ont fait exploser mon cerveau en fin de journée 🤯
```suggestion
  it('picks only challenge on certifiable competences', async () => {
```
Je suis même pas sûre que ce que je propose est bien équivalent xD